packages/frontend/src/components/Explorer/VisualizationCard/VisualizationSidebar.tsx

```diff
@@ -22,24 +23,25 @@ const VisualizationSidebar: FC<{
     chartType: ChartType;
     savedChart?: SavedChart;
     isProjectPreview?: boolean;
-    isEditingDashboardChart?: boolean;
     isOpen: boolean;
     onClose: () => void;
     onOpen: () => void;
 }> = memo(
-    ({
-        chartType,
-        savedChart,
-        isProjectPreview,
-        isEditingDashboardChart,
-        isOpen,
-        onOpen,
-        onClose,
-    }) => {
-        const sidebarVerticalOffset =
-            (isProjectPreview && !isEditingDashboardChart ? 35 : 0) + // Preview header
```

Comment:
we actually don't need to check if it's editing a dashboard chart to apply the correct styles.

Reply:
Nice. The new version is much better.